xattrvi

```diff
@@ -119,9 +120,14 @@ with open(tmpfile.name, 'w') as f:
 
 
 # let the user change those attributes
-retval = subprocess.call([os.environ.get('EDITOR') or 'nano', tmpfile.name])
+if re.findall('gvim',args.editor):
+    # needs special nofork for subprocess to wait for gvim to return
+    retval = subprocess.call(['gvim','--nofork', tmpfile.name])
+else:
+    retval = subprocess.call([args.editor or os.environ.get('EDITOR') or 'nano', tmpfile.name])
```

Comment:
With a default value for editor, we can just call `args.editor.split()` here, making this much simpler.